docs/development/virtual_environments.rst

```diff
@@ -129,10 +129,126 @@ virtualized, rather than running on hardware. You will need to
 playbook from running with Vagrant-specific info.
 
 You can provision production VMs from an Admin Workstation (most realistic),
-or from your host. Instructions for both follow.
+or from your host. If your host OS is Linux-based and you plan to use an Admin 
+Workstation, you will need to switch Vagrant's default virtualization provider 
+from ``virtualbox`` to  ``libvirt``.  The Admin Workstation VM configuration 
+under Linux uses QEMU/KVM, which cannot run simultaneously with Virtualbox.
+
+Instructions for both installation methods follow.
 
 .. _prod_install_from_tails:
 
+Switch Vagrant provider to libvirt - Ubuntu 16.04
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Uninstall any exiting version of Vagrant. Install libvirt and QEMU:
+
+.. code:: sh
+
+   sudo apt-get update
+   sudo apt-get remove vagrant
+   sudo apt-get install libvirt-bin libvirt-dev qemu-utils qemu
+   sudo /etc/init.d/libvirt-bin restart
+
+Add your user to the libvirtd group:
+
+.. code:: sh
+
+   sudo addgroup libvirtd
+   sudo usermod -a -g libvirtd $USER
+
+Log out, then log in again. Verify that libvirt is installed and KVM is
+available:
+
+.. code:: sh
+
+   libvirtd --version 
+   kvm-ok
+
+Reinstall Vagrant and install the required plugins for converting and using 
+libvirt boxes:
+
+.. code:: sh
+   
+   wget https://releases.hashicorp.com/vagrant/1.9.1/vagrant_1.9.1_x86_64.deb
+   sudo dpkg -i vagrant_1.9.1_x86_64.deb
```

Comment:
_Reinstalling_ vagrant shouldn't be necessary here; just adding the plugins, below. Was this required on your setup?

Reply:
It was suggested in docs I read, but it may not be necessary. Updating docs to reflect that.

Reply:
We don't need to reference vagrant installation here at all, since that's already covered in the `setup_development.rst` document. Perhaps a cross-link would be useful, but no more.